src/components/App.tsx

```diff
@@ -240,8 +240,9 @@ type PointerDownState = Readonly<{
     // This is determined on the initial pointer down event
     wasAddedToSelection: boolean;
     // Whether selected element(s) were duplicated, might change during the
-    // pointer interation
+    // pointer interaction
     hasBeenDuplicated: boolean;
+    hasHitCommonBoundingBoxOfSelectedElements: boolean;
```

Comment:
I think we don't need this state and instead inline the detection in the two places you read it. In one place you're already reading it just a few lines above.

Reply:
I started with that approach. But that leads to a bug. If we move the cursor fast, and for a moment, it ends up not being inside the common bounding box, it will deselect the elements. So I needed a way to check if the pointer down hit the common bounding box or not. In particular, I needed that `onPointerMoveFromPointerDownHandler()` since the event I had access to was the pointer moving and not the pointer down. That's why I added that state.

Reply:
You're reading it only in pointer down events, not during move, so that doesn't make sense to me.

In fact here the read can technically lead to incorrect behavior (not in practice right now) because unlike the expression above, you're not checking the `pointerDownState.hit.hasHitCommonBoundingBoxOfSelectedElements` doesn't relate to different elements the you're clicking:

https://github.com/Jnforja/excalidraw/blob/ae7b278a5b81ea31bb13223166c9d685d9e533e9/src/components/App.tsx#L2902-L2904

Reply:
I'm sorry, I'm not entirely sure what you're suggesting we change on `onPointerMoveFromPointerDownHandler()`. Is it that we change L2904 from 
`pointerDownState.hit.hasHitCommonBoundingBoxOfSelectedElements` to
```
this.isHittingCommonBoundingBoxOfSelectedElements(
          pointerCoords,
          getSelectedElements(this.scene.getElements(), this.state),
        )
```
is that it?

Reply:
I actually misread it, and it indeed is inside a move handler so I can see what you meant. Never mind :(.

Reply:
No problem :)